third_party/tsl/tsl/platform/default/rocm_rocdl_path.cc

```diff
@@ -40,6 +40,13 @@ string RocmRoot() {
 #endif
 }
 
-string RocdlRoot() { return io::JoinPath(RocmRoot(), "amdgcn/bitcode"); }
+string RocdlRoot() {
+  if (const char* device_lib_path_env = std::getenv("HIP_DEVICE_LIB_PATH")) {
+   return device_lib_path_env;
+  }
+  else{
+   return io::JoinPath(RocmRoot(), "amdgcn/bitcode");
+  }
```

Comment:
Can you run clang-format on this? I believe the internal formatter will complain about this formatting once we import it.

Reply:
sharp-eyed!